File: poshub-spa/src/app/dtos/catalogImportEntityDto.ts
import { ProductDto } from "./productDto";

export class CatalogImportEntityDto {
  constructor(
    // public location: LocationDto = new LocationDto(),
    // public categories: CategoryDto[] = [],
    public products: ProductDto[] = [],
    // public modifierGroups: ModifierGroupDto[] = [],
    // public modifiers: ModifierDto[] = []
  ) {}
}

File: PosHubApi/Dtos/CatalogImportEntityDto.cs
namespace PosHubApi.Dtos
{
    public class CatalogImportEntityDto
    {
        public LocationHoursDto Location { get; set; }
        public List<CategoryDto> Categories { get; set; }
        public List<ProductDto> Products { get; set; }
        public List<ModifierGroupDto> ModifierGroups { get; set; }
        public List<ModifierDto> Modifiers { get; set; }
        public string ErrorMessage  { get; set; } = "";
    }
}